src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -358,3 +358,60 @@ def test_synthetic_to_openai_completions(self, default_tokenizer):
 
         assert pa_json is not None
         assert len(pa_json["data"]) == 5
+
+    @pytest.mark.parametrize(
+        "output_format",
+        [
+            (OutputFormat.OPENAI_CHAT_COMPLETIONS),
+            (OutputFormat.OPENAI_COMPLETIONS),
+            (OutputFormat.TRTLLM),
+            (OutputFormat.VLLM),
+        ],
+    )
+    def test_llm_inputs_extra_inputs(self, default_tokenizer, output_format) -> None:
+        # Simulate --request-input arguments
+        request_inputs = {"additional_key": "additional_value"}
+
+        # Generate input data with the additional request inputs
+        pa_json = LlmInputs.create_llm_inputs(
+            input_type=PromptSource.SYNTHETIC,
+            output_format=output_format,
+            num_of_output_prompts=5,  # Generate a small number of prompts for the test
+            add_model_name=False,
+            add_stream=True,
+            tokenizer=default_tokenizer,
+            extra_inputs=request_inputs,  # Pass the simulated --request-input arguments here
+        )
+
+        assert len(pa_json["data"]) == 5
+
+        if (
+            output_format == OutputFormat.OPENAI_CHAT_COMPLETIONS
+            or output_format == OutputFormat.OPENAI_COMPLETIONS
+        ):
```

Comment:
parentheses are necessary here?

Reply:
Good catch. Old C++ habit. Removed!

Reply:
Frinkin C++ :shakes_fist:

Reply:
Looks like the pre-commit hooks are actually adding this. Perhaps for readability because it's a multi-line if statement.

Reply:
okay. atleast we have a reason and know that it will be mostly consistent in the code base.
thank you